src/Blocks/components/form/assets/state-init.js

```diff
@@ -55,6 +55,7 @@ export const StateEnum = {
 	SAVE_AS_JSON: 'saveAsJson',
 	IS_ADMIN: 'isAdmin',
 	IS_USED: 'isUsed',
+	IS_USED_LOCALSTORAGE: 'isUsedLocalStorage',
```

Comment:
```suggestion
	IS_LOCAL_STORAGE_USED: 'isLocalStorageUsed',
```

Reads better.

Affects all the code where the enum is being used.

Reply:
All enums are written in he similar way so I will not be changing this, tnx for the suggestion